### PR TITLE
PMM-405 Fixed error message for explain on empty queries

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

**Review by roman-vynar — APPROVED** [review]
🚢

File: query/mysql/executor.go [hunk 1]
```diff
@@ -178,7 +178,10 @@ func (e *QueryExecutor) classicExplain(tx *sql.Tx, query string) (classicExplain
 	// Partitions are introduced since MySQL 5.1
 	// We can simply run EXPLAIN /*!50100 PARTITIONS*/ to get this column when it's available
 	// without prior check for MySQL version.
-	rows, err := tx.Query(fmt.Sprintf("EXPLAIN /*!50100 PARTITIONS*/ %s", query))
+	if strings.TrimSpace(query) == "" {
+		return nil, fmt.Errorf("cannot run EXPLAIN on an empty query example")
+	}
+	rows, err := tx.Query(fmt.Sprintf("EXPLAIN PARTITIONS %s", query))
```
Comment (roman-vynar) [inline query/mysql/executor.go]: 5.7:
```
| Warning | 1681 | 'PARTITIONS' is deprecated and will be removed in a future release. |
```

Do we need that `PARTITIONS` at all?

Comment (roman-vynar) [inline query/mysql/executor.go]: 5.7:

```
| Warning | 1681 | 'PARTITIONS' is deprecated and will be removed in a future release. |
```

Do we need that `PARTITIONS` at all?

Comment (percona-csalguero) [reply query/mysql/executor.go]: Removed

**Pull request merged**